src/pineko/theory.py

```diff
@@ -1,10 +1,19 @@
 # -*- coding: utf-8 -*-
+"""Tools related to generation of a list of FK tables.
+
+The typical use case of pineko is the generation of a list of FK tables,
+all with common theory parameters. The collective list of this FK tables
+together with other theory ingredients (such as C-factors) are often
+commenly refered to as 'theory'.
+"""
 import logging
 import time
 
 import eko
+import numpy as np
 import rich
 import yaml
+from eko import strong_coupling as sc
```

Comment:
Maybe would be better to import eko only once but I don't know, just wondering

Reply:
`import eko` is not enough, because `eko.strong_coupling` would not be defined (for that to happen, we need to `import strong_coupling` in `__init__.py`, that is not the case - and most likely we don't want to, at the moment, since we want to directly expose only a minority of things that we believe the user is going to use more often).

You'd need at least:
```py
import eko.strong_coupling
```
Not much different.

Reply:
If you refer to import it multiple times in the package, that is not a problem (if a module is already loaded at runtime, Python won't load it again - modules are runtime singletons).

Reply:
Yes I was thinking to use just `import eko` and then use the function as `eko.strong_coupling.whatever_func`. However,  if importing twice is not a performance problem as you said, it is just fine to import twice.